File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    answer = ""
    arr = ["zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
    
    for idx, val in enumerate(arr):
        if val in s:
            s = s.replace(val, str(idx))
        answer = s
    
    return int(answer)

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
import java.util.List;

class Solution {
    public int solution(String s) {
        int answer = 0;

        List<String> list = List.of("zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine");
        for (int i = 0; i < list.size(); i++) {
            s = s.replace(list.get(i), String.valueOf(i));
        }

        answer = Integer.parseInt(s);

        return answer;
    }
}